helmfile/charts/networkpolicy-service/values.yaml

```diff
@@ -14,6 +14,28 @@ additional: {}
 #         foo: bar
 #     ingress: {}
 #     egress: {}
+global:
+  apiserver:
+    ips:
+      - "0.0.0.0/0"
+    port: 6443
+  scNodes:
+    ips:
+      - "0.0.0.0/0"
+  wcIngress:
+    ips:
+      - "0.0.0.0/0"
+  objectStorage:
+    ips:
+      - "0.0.0.0/0"
+    ports:
+      - 443
+  ingressUsingHostNetwork: false
+  externalLoadBalancer:
+    enabled: false
+    ips:
+      - "0.0.0.0/0"
```

Comment:
This should be the same as `scIngress` so let's move that to `common-config` and use that instead

Reply:
Yeah, whenever `externalLoadBalancer.enabled: true` they should be the same. I'll refactor that.